tests/test_aaa_server_group.rb

```diff
@@ -231,246 +185,227 @@ def test_aaaservergroup_collection_multi_tacacs
     aaa_group3 = AaaServerGroup.new(group_name3, :tacacs)
 
     groups = AaaServerGroup.groups(AAA_SERVER_GROUP_TACACS_SERVER)
-    refute_empty(groups, "Error: AaaServerGroup collection is not filled")
+    refute_empty(groups, 'Error: AaaServerGroup collection is not filled')
     assert_equal(3, groups.size,
-                 "Error: AaaServerGroup collection not reporting correct size")
+                 'Error: AaaServerGroup collection not reporting correct size')
     assert(groups.key?(group_name1),
-                 "Error: AaaServerGroup collection does contain #{group_name1}")
+           "Error: AaaServerGroup collection does contain #{group_name1}")
     assert(groups.key?(group_name2),
-                 "Error: AaaServerGroup collection does contain #{group_name2}")
+           "Error: AaaServerGroup collection does contain #{group_name2}")
     assert(groups.key?(group_name3),
-                 "Error: AaaServerGroup collection does contain #{group_name3}")
+           "Error: AaaServerGroup collection does contain #{group_name3}")
 
     detach_aaaservergroup(aaa_group1)
     detach_aaaservergroup(aaa_group2)
     detach_aaaservergroup(aaa_group3)
   end
 
-  def test_aaaservergroup_collection_multi_radius
+  def test_collection_multi_radius
     # TBD
   end
 
-  def test_aaaservergroup_servers_tacacs
+  def test_servers_tacacs
     clean_tacacs_config
-    server_name1 = "server1"
-    server_name2 = "server2"
+    server_name1 = 'server1'
+    server_name2 = 'server2'
     server1 = create_tacacsserverhost(server_name1)
     server2 = create_tacacsserverhost(server_name2)
 
-    aaa_group = AaaServerGroup.new("Group1", :tacacs)
+    aaa_group = AaaServerGroup.new('Group1', :tacacs)
 
     # pre check that default servers are empty
     default_server = AaaServerGroup.default_servers
-    assert_empty(default_server, "Error: Default Servers are not empty")
+    assert_empty(default_server, 'Error: Default Servers are not empty')
 
     aaa_group.servers = [server_name1, server_name2]
 
     # Check collection size
     servers = aaa_group.servers.keys
-    assert_equal(2, servers.size(),
-                 "Error: Collection is not two servers")
+    assert_equal(2, servers.size,
+                 'Error: Collection is not two servers')
     assert(servers.include?('server1'),
-                 "Error: Collection does not contain #{server_name1}")
+           "Error: Collection does not contain #{server_name1}")
     assert(servers.include?('server2'),
-                 "Error: Collection does not contain #{server_name2}")
+           "Error: Collection does not contain #{server_name2}")
 
     detach_aaaservergroup(aaa_group)
     detach_tacacsserverhost(server1)
     detach_tacacsserverhost(server2)
   end
 
-  def test_aaaservergroup_servers_radius
+  def test_servers_radius
```

Comment:
Still TBD. This commit was solely a rubocop cleanup to get the branch happy with CI.

I have added a US to Scrum 3's backlog to flesh these out.